data/governments.txt

```diff
@@ -516,8 +530,25 @@ government "Hai (Friendly Unfettered)"
 		"Pirate (Devil-Run Gang)" -.01
 		"Korath" -.01
 		"Hai" .01
+		"Hai (Wormhole Access)" .01
+		"Hai (Unfettered)" 1
```

Comment:
Is this government ever used at a time where the Unfettered are hostile to me?

Reply:
Yes, naturally. It is only used during HR and the rep is reset at the start.
If you kill that many Unfettered in the middle of a Hai-human crisis on the brink of war... Screw that player!

Reply:
Also its an unfettered gov, just done this way so its easier for the story, but that means they ought to care.